### DISCUSSION TITLE
About pre-trained model

### DISCUSSION THREAD

**Discussion by sys1874** [snap-stanford/ogb#125] [General]
Hi!   Can we use the pre-trained model like bert, roberta, XL-net as our warmup model?
Thanks

**Comment by weihua916** [comment]
Hi! For OGB-LSC, you cannot. Also, we did not provide the raw text for the datasets. We've already used roberta to obtain the features for you.

**Reply by joneswong** [reply]
BTW, for the graph-level track, is it allowed to apply unsupervised pre-training with a dataset including the unlabeled test molecules. According to the rules declared in the kddcup website, I think it is not allowed. However, how do you check this?

**Reply by weihua916** [reply]
Good point! No, it is not allowed to use external molecules. We can check this based on your code, which you need to submit once you win the contest.
